glue/core/roi.py

```diff
@@ -74,6 +74,20 @@ def contains(self, x, y):
         """
         raise NotImplementedError()
 
+    def contains3d(self, x, y, z):
+        """Return true/false for each x/y/z pair.
+
+        :param x: Array of X locations
+        :param y: Array of Y locations
+        :param y: Array of Z locations
+
+        :returns: A Boolean array, where each element is True
+                  if the corresponding (x,y,z) tuple is inside the Roi.
+
+        :raises: UndefinedROI exception if not defined
```

Comment:
Could you use Numpydoc format here? I'm trying to transition the docstrings to numpydoc progressively, so it would be good to avoid adding new code that's not in Numpydoc.